### PR TITLE
New sections for Ceph point releases (gh#SUSE/doc-ses#308)

### LINKED ISSUE #308: SES6 Deployment Guide: Add new sections to Appendix "A Ceph Maintenance Updates Based on Upstream 'Nautilus' Point Releases"
**Link To Section**

https://documentation.suse.com/ses/6/single-html/ses-deployment/#id-1.4.7

**Summary**

The last upstream Nautilus release mentioned in the appendix is 14.2.5. We need to add sections for 14.2.6, 14.2.7, 14.2.8, and 14.2.9

**Text of the New Sections**

14.2.6

This release fixed a ceph-mgr bug that caused mgr becoming unresponsive on larger clusters. SES users were never exposed to the bug.

14.2.7

This upstream release patched two security flaws:

* CVE-2020-1699: a path traversal flaw in Ceph dashboard that could allow for potential information disclosure
* CVE-2020-1700: a flaw in RGW beast frontend that could lead to denial of service from an unauthenticated client

In SES6, these flaws were patched in ceph 14.2.5.382+g8881d33957b.

14.2.8

In addition to bugfixes, this major upstream release brought a number of notable changes:

* The default value of `bluestore_min_alloc_size_ssd` has been changed to 4K to improve performance across all workloads.

* The following OSD memory config options related to bluestore cache autotuning can now
be configured during runtime:

        osd_memory_base (default: 768 MB)
        osd_memory_cache_min (default: 128 MB)
        osd_memory_expected_fragmentation (default: 0.15)
        osd_memory_target (default: 4 GB)

    The above options can be set with:

        ceph config set osd <option> <value>

* The MGR now accepts `profile rbd` and `profile rbd-read-only` user caps.
These caps can be used to provide users access to MGR-based RBD functionality
such as `rbd perf image iostat` and `rbd perf image iotop`.

* The configuration value `osd_calc_pg_upmaps_max_stddev` used for upmap
balancing has been removed. Instead, use the mgr balancer config
`upmap_max_deviation` which now is an integer number of PGs of deviation
from the target PGs per OSD. This can be set with a command like
`ceph config set mgr mgr/balancer/upmap_max_deviation 2`. The default
`upmap_max_deviation` is 5. There are situations where crush rules
would not allow a pool to ever have completely balanced PGs. For example, if
crush requires 1 replica on each of 3 racks, but there are fewer OSDs in 1 of
the racks. In those cases, the configuration value can be increased.

* CephFS: multiple active MDS forward scrub is now rejected. Scrub currently
only is permitted on a file system with a single rank. Reduce the ranks to one
via `ceph fs set <fs_name> max_mds 1`.

* Ceph now refuses to create a file system with a default EC data pool. For
further explanation, see:
https://docs.ceph.com/docs/nautilus/cephfs/createfs/#creating-pools

* Ceph will now issue a health warning if a RADOS pool has a `pg_num` value that is not a power of two. This can be fixed by adjusting the pool to a nearby power of two:

        ceph osd pool set <pool-name> pg_num <new-pg-num>

    Alternatively, the warning can be silenced with:

        ceph config set global mon_warn_on_pool_pg_num_not_power_of_two false

14.2.9

This upstream release patched two security flaws:

* CVE-2020-1759: Fixed nonce reuse in msgr V2 secure mode
* CVE-2020-1760: Fixed XSS due to RGW GetObject header-splitting

In SES6, these flaws were patched in ceph 14.2.5.389+gb0f23ac248.

**Notes**

The new sections should be added to the top of the appendix, and in reverse order, to preserve the existing reverse ordering of the sections.

### PR DESCRIPTION
**Pull request by tbazant**
Adds entries for new Ceph point releases.
Closes #308

### CODE REVIEW

File: xml/ceph_maintenance_updates.xml [hunk 1]
```diff
@@ -33,6 +33,125 @@
...
+    where crush rules would not allow a pool to ever have completely balanced
+    PGs. For example, if crush requires 1 replica on each of 3 racks, but there
+    are fewer OSDs in 1 of the racks. In those cases, the configuration value
+    can be increased.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    &cephfs;: multiple active &mds; forward scrub is now rejected. Scrub
+    currently only is permitted on a file system with a single rank. Reduce the
```
Comment (lproven) [inline xml/ceph_maintenance_updates.xml:111]:
```suggestion
    &cephfs;: multiple active &mds; forward scrub is now rejected. Scrub is
    currently only permitted on a file system with a single rank. Reduce the
```

File: xml/ceph_maintenance_updates.xml [hunk 2]
```diff
@@ -33,6 +33,125 @@
...
+    ranks to one via <command>ceph fs set <replaceable>FS_NAME</replaceable>
+    max_mds 1</command>.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    &ceph; will now issue a health warning if a &rados; pool has a
+    <option>pg_num</option> value that is not a power of two. This can be fixed
+    by adjusting the pool to a nearby power of two:
```
Comment (lproven) [inline xml/ceph_maintenance_updates.xml:120]:
```suggestion
    by adjusting the pool to an adjacent power of two:
```

Comment (lproven) [reply xml/ceph_maintenance_updates.xml:120]: I am not sure about this; it is a guess.

"Nearby" could mean 1-2 away, which could be a very big step! "Adjacent" implies the next nearest, i.e. rounding up or rounding down, which would seem to make more sense to me.

Comment (smithfarm) [reply xml/ceph_maintenance_updates.xml:120]: The upstream release notes say "nearby", but I think "adjacent" is OK, too.

**Pull request merged**